extensions/sql-migration/src/models/loginMigrationModel.ts

```diff
@@ -26,6 +26,13 @@ export enum LoginMigrationStep {
 	MigrationCompleted = 3,
 }
 
+export enum LoginMigrationErrors {
+	CollectingSourceLoginsFailed,
+	CollectingTargetLoginsFailed,
+	ConnectingToTargetFailed,
```

Comment:
defining these items as strings will simplify the code the references them:  
ex.  CollectingSourceLoginsFailed = 'CollectingSourceLoginsFailed '...